rock_paper_scissors.py

```diff
@@ -0,0 +1,44 @@
+from random import randint
+from sys import exit
+
+RPS_LIST = ["rock", "paper", "scissors"]
+CHOICES_DICT = {"r": "rock", "p": "paper", "s": "scissors", "q": "quit"}
+USER_NAME = input("Type Players Name:")
+
+while True:
+    raw_user_choice = input("\n\n Hey " + USER_NAME + ", please choose r, p, or s:")
+
+    # let them quit
+    if raw_user_choice == 'q':
+        print('It was nice beating you')
+        exit()
+    
+    if raw_user_choice not in CHOICES_DICT:
+        print("WTF dude...\n\n")
+        continue
+        
+    user_choice = CHOICES_DICT[raw_user_choice]  # r -> rock
+    cpu_choice = RPS_LIST[randint(0, 2)]    # 0 -> rock
+
+    print('You chose "' + user_choice + '" and the computer chose "' + cpu_choice + '"')
```

Comment:
There is a really nice way to do this kind of string formatting in python3, but it only works in python3. We should switch to that and tell them that have to use python 3 in the readme.

The new syntax is this this:
`f'You chose "{user_choice}" and the computer chose "{cpu_choice}"'`

Things to note:
- there is a sneaky little `f` before the string
- it is now all one big string instead of many strings added together
- any time there is a `{ some_variable }` in the string, it looks for whatever variable is in between them

Reply:
Actually, we probably want to define all of the strings as constants, at the top of the file, just under the imports. We will need to use a different kind of string formatting for that to work because f-strings look for variables in the scope.

It will end up looking like this at the top:
`RESULT_SUMMARY = 'You chose "{user_choice}" and the computer chose "{cpu_choice}"'`

And we will format it like this:
`RESULT_SUMMARY.format(user_choice=user_choice, cpu_choice=cpu_choice)`

It's okay if this is confusing. I'll explain more in person.